modules/countdown.js

```diff
@@ -4,21 +4,52 @@ require('moment-countdown')
 let timeout = null
 
 const ARTICLE_50 = '2020-01-31T00:00:00+01:00'
+const TRANSITION = '2020-12-31T00:00:00+01:00'
 
 function autoCount (bot, lastTick) {
-  const thisTick = moment(ARTICLE_50).countdown().toString().split(/, | and /)[0]
+  const now = moment()
+
+  var eventName
+  var deadline
+  if (moment(ARTICLE_50).isBefore(now)) {
+    eventName = 'Article 50'
+    deadline = ARTICLE_50
+  } else {
+    eventName = 'The transition period'
+    deadline = TRANSITION
+  }
+
+  const thisTick = moment(deadline).countdown().toString().split(/, | and /)[0]
+
   if (lastTick != null && thisTick !== lastTick) {
-    bot.broadcast('Article 50 expires in ' + lastTick)
+    bot.broadcast(`${eventName} expires in ${lastTick}`)
   }
+
   timeout = setTimeout(autoCount, 1000, bot, thisTick)
 }
 
+function expiry (name, date) {
+  const now = moment()
+  const expiryDate = moment(date)
+  const countdown = expiryDate.countdown().toString()
+
+  if (expiryDate.isBefore(now)) {
+    return `${name} expired ${countdown} ago`
+  }
+  return `${name} expires in ${countdown}`
+}
+
 module.exports = {
   commands: {
     a50: {
       help: 'Gets the time until the UK Article 50 procedure expires',
       command: function () {
-        return 'Article 50 expires in ' + moment(ARTICLE_50).countdown().toString()
+        const now = moment()
+
+        if (moment(ARTICLE_50).isBefore(now)) {
```

Comment:
You've duplicated this code further up; would it be better to move this into a function that both bits call? You'd then be able to move the constants at the top into that function.